manifests/init.pp

```diff
@@ -173,8 +170,7 @@
   String $keystore_file = $candlepin::params::keystore_file,
   Optional[String] $keystore_password = $candlepin::params::keystore_password,
   String $keystore_type = $candlepin::params::keystore_type,
-  String $truststore_file = $candlepin::params::truststore_file,
-  Optional[String] $truststore_password = $candlepin::params::truststore_password,
+  Optional[String] $truststore_file = $candlepin::params::truststore_file,
```

Comment:
Is this an absolute path or can we accept a relative one? A question I don't know the answer to, but it looks like we now use an absolute path.
```suggestion
  Optional[Stdlib::Absolutepath] $truststore_file = $candlepin::params::truststore_file,
```